src/sonic-yang-models/yang-models/sonic-mux-cable.yang

```diff
@@ -0,0 +1,93 @@
+module sonic-mux-cable {
+
+    yang-version 1.1;
+
+    namespace  "http://github.com/Azure/sonic-mux-cable";
+
+    prefix mux-cable;
+
+    import ietf-inet-types {
+        prefix inet;
+    }
+
+    import sonic-port {
+        prefix port;
+    }
+
+    description "MUX CABLE YANG module for SONiC OS";
+
+    revision 2022-08-10 {
+        description "Initial version";
+    }
+
+    container sonic-mux-cable {
+
+        container MUX_CABLE {
+
+            description "MUX_CABLE part of config_db.json";
+
+            leaf PEER_TOR {
```

Comment:
Peer_tor is not part of the MUX_CABLE table but part of PEER_SWITCH table. Please remove from here

Reply:
Thx Prince. 
Kindly also suggest if any condition is missing from yang models. For example, say, if SERVER ipv4 address should be unique across PORTS.

Reply:
Yes, thats an implicit assumption. But  agree with you.. we can make it part of the yang.